examples/cpu/serving/triton/model_utils/config.pbtxt

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+max_batch_size: 32
+backend: "python"
+input [
+  {
+    name: "INPUT0"
+    data_type: TYPE_INT64
+    dims: [ -1 ]
+  }
+]
+output [
+  {
+    name: "OUTPUT0"
+    data_type: TYPE_FP32
+    dims: [-1 ]
+  }
+  
+  ]
+instance_group [
+  {
+    count: 1
+    kind: KIND_CPU
+  }
+]
```

Comment:
Do we support multi-instances? If so, how are the CPU cores allocated to each instance and how are the threads affinitized?

Reply:
To run this example on multiple instances, users need to manually initiate the Docker container on each host instance using './start.sh host.' Additionally, for a distributed setup, it's recommended to incorporate a load balancer between the client and multiple hosts. As for now that's not the part of the example. The CPU core allocation and thread affinitization are handled within the './start.sh' script, specifically in lines 42-43.

Reply:
Can we add how to run multiple instances in the README doc?

Reply:
As previously mentioned, incorporating a load balancer between the client and host is necessary, but it hasn't been implemented yet. The README provides instructions on how to run it in a supported scenario: on two separate instances (one for the client and one for the host).